web/modules/custom/asset_release/config/install/core.entity_view_display.rdf_entity.asset_release.default.yml

```diff
@@ -36,6 +33,25 @@ targetEntityType: rdf_entity
 bundle: asset_release
 mode: default
 content:
+  created:
+    weight: 1
+    label: hidden
```

Comment:
The creation date is not actually visible in the frontend, I think it is probably hidden in the theme layer. Let's make sure they are also hidden in the form display configuration, since this was causing trouble in ISAICP-3832.

Reply:
The creation date is actually used in the page title template of the asset release canonical page. The frontend team is using this configuration to render it there, and they don't show it in the normal page template.